src/composables/transactionTx.ts

```diff
@@ -48,14 +48,14 @@ export function useTransactionTx({
 
   const isAllowance = computed((): boolean => (
     !!innerTx.value
-    && FUNCTION_TYPE_DEX.allowance.includes(innerTx.value.function as TxFunctionRaw)
+    && FUNCTION_TYPE_DEX.allowance.includes(innerTx?.value?.function as TxFunctionRaw)
```

Comment:
those question marks are not necessary as you check it in the first line `!!innerTx.value`